src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,25 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Note that static imports are to be ordered above normal imports, contrary to the default settings in IntelliJ.  
See: https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements  

![image](https://user-images.githubusercontent.com/66059643/131636462-bd0e6583-a1b5-426d-97b1-cf45fe6aeda8.png)